src/schoology-extractor/tests/test_lib/test_request_client.py

```diff
@@ -0,0 +1,231 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+import pytest
+
+from data_extractor.lib.request_client import RequestClient
+
+pytest.fake_key = 'TEST_KEY'
```

Comment:
Do these have to be added to the pytest module?  I'd think they are fine standalone.
Separately, Pylance doesn't like them because they aren't part of the module definition.

Reply:
You are right, I removed the pytest part and it worked.